### ISSUE TITLE
[Abandoned Plugin] OctoPrint-Thingiverse

### ISSUE THREAD

**Issue by GuySandler** [OctoPrint/plugins.octoprint.org#1082]
### Which Plugin?

https://plugins.octoprint.org/plugins/thingiverse/

### Link to ticket on plugin's issue tracker confirming abandonment

https://github.com/stefancandrea/OctoPrint-Thingiverse/blob/master/README.md

### Is there someone open to adopting it?

I tried but then realized it's kind of a security risk and just skips 1 step

**Comment by ender-bit** [comment]
ill take over if you want me to

**Comment by jacopotediosi** [comment]
@foosel @stefancandrea 

I feel that this plugin should be removed and/or blacklisted, as it appears to be very dangerous.

At some point after the plugin’s initial release, the Thingiverse website introduced security policies to prevent the site from being embedded in iframes. Since this change made the plugin unusable, the author updated the plugin listing on [plugins.octoprint.org](https://plugins.octoprint.org/plugins/thingiverse/), as well as the settings tab and README, suggesting users to install third-party browser extensions and configure them to disable fundamental browser security protections such as the [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/CSP).

These configuration changes are dangerous and — if applied [following the plugin author instructions](https://github.com/stefancandrea/OctoPrint-Thingiverse/blob/7ee030d65f6e2629ad581014db8dfe9ec867f233/octoprint_thingiverse/templates/thingiverse_settings.jinja2#L42C82-L42C142) — are applied globally, meaning that users will be disabling these protections not only for OctoPrint but for all websites they visit.

![Screenshot of the settings tab](https://github.com/user-attachments/assets/1a8a54af-419f-4c8f-8794-3ad9ad767838)

The issue with this plugin is at its root. It needs to be completely rewritten using the Thingiverse API.

Attempting to embed the Thingiverse website in an iframe, or trying to bypass the `X-Frame-Options` security protection — as seen in [this commit](https://github.com/stefancandrea/OctoPrint-Thingiverse/commit/3a741975b2e6fcc5898bc5e7cd6c6b4edce64b1b) — are not viable solutions given how modern browsers work and the current standards for web security.

**Comment by foosel** [comment]
Agreed. I've just removed it (this time also including the assets I hope)
